maestro-test/src/test/resources/051_set_location.yaml

```diff
@@ -0,0 +1,7 @@
+appId: com.example.app
+---
+- launchApp:
+    appId: com.example.app
+- setLocation:
+    lat: 12.5266
```

Comment:
Let's use `latitude` and `longitude` instead of shortened `lat`, `long`

Reply:
Done